src/Achievements/Achievements.csproj

```diff
@@ -1,12 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
-  </PropertyGroup>
+<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Wetonomy.TokenSystem" Version="0.0.0" />
```

Comment:
We should maybe revert these. To satisfy the dependencies we can use local Nuget packages.